Project4_election_scraper/README

```diff
@@ -0,0 +1,14 @@
+Write a script that downloads election results of every municipality in every district into csv file.
+
+At this page, you first have to choose the district by the X key in the column Výber obce (on the right).
+So for when you choose i.e. district Příbram you then have to choose the municipality.
+This time you choose by the number of the municipality (column číslo on the left).
+So, i.e. Dobříš will have the number 540111.
+
+Input
+The script expects two inputs:
+
+Link where the list of the municipalities is located
+name of the csv file without the suffix .csv
+Output
+The header of the csv file should have the following: municipality code, name of location, voters in the list, distributed voting envelopes, valid votes, candidate parties. Each row of the csv file will then represent individual municipalities. So an example of Hradec Králové would look like this:
```

Comment:
Great description. Could you also add an example of running this script (just the script with command line arguments) so a user can try it right away?